### PR TITLE
Add dark mode and minor css fixed

### PR DESCRIPTION
**Pull request by bajran**
Added Dark mode Support -

1] On clicking on the toggle button it will change the mode i.e ( text color/ background color etc) of UI 
2] It will persist the selected mode as we are storing the mode value in local storage

### CODE REVIEW

**Comment by bajran** [comment]
Added Dark mode Support -

1] On clicking on the toggle button it will change the mode i.e ( text color/ background color etc) of UI 
2] It will persist the selected mode as we are storing the mode value in local storage

**Comment by PrasadG193** [comment]
@bajran can we add icons for the dark mode button? Something like - https://docs.docker.com/ have

**Comment by PrasadG193** [comment]
Also, the preview can be viewed at: https://deploy-preview-43--kyaml2go.netlify.app/

**Comment by PrasadG193** [comment]
Can we also change the input box colors for CR?
image

**Comment by bajran** [comment]
> Can we also change the input box colors for CR?
> image

yes will update this

**Comment by bajran** [comment]
> @bajran can we add icons for the dark mode button? Something like - https://docs.docker.com/ have

will add the icon @PrasadG193 thanks for suggestion

File: frontend/style.css [hunk 1]
```diff
@@ -50,59 +94,83 @@ html, body, main {
...
-.fa-copy:hover{
+/**
+* Removed hover on copy functionality
+* It doesn't look good
+*/
+/* .fa-copy:hover {
   transform: scale(1.3);
-}
-#codecopied{
+} */
```
Comment (PrasadG193) [inline frontend/style.css:145]: Please remove commented section if not needed

**Review by PrasadG193 — APPROVED** [review]

**Pull request merged**